src/modules/auth/header.js

```diff
@@ -41,11 +41,15 @@ function HeaderComponent(props) {
 
 	// Get access control URL
 	let access_control_url = window.location.protocol + '//' + window.location.host + window.location.pathname + '#/auth/access-control';
-	// Check if Tenant service is available to get the access control URL with tenant
+
+	// Get URL of About
+	let about_url = window.location.protocol + '//' + window.location.host + window.location.pathname + '#/about';
+	// Check if Tenant service is available to get the access control URL and about with tenant
 	if (App.Services.TenantService) {
 		let currentTenant = App.Services.TenantService.get_current_tenant();
 		if (currentTenant) {
 			access_control_url = window.location.protocol + '//' + window.location.host + window.location.pathname + '?tenant=' + currentTenant + '#/auth/access-control';
+			about_url = window.location.protocol + '//' + window.location.host + window.location.pathname + '?tenant=' + currentTenant + '#/auth/about';
```

Comment:
I believe you don't need to build the full (absolute) URL here.
`#/auth/about` part aka `<Link to={} />` will be much better.

Reply:
I tried to use <Link to...> the following way:
```
<DropdownItem tag="div" >
	<Link to="/auth/about">
		{t('AuthHeaderDropdown|About')}
	</Link>
</DropdownItem>
```

But it requires some additional styling for usability, i'm attaching photos to show the problem. would you like me to make these additional styling?
![with-Link](https://user-images.githubusercontent.com/57274895/123248503-4b8d8880-d4e8-11eb-8b38-d2ec57320390.png)
![with-link-hover](https://user-images.githubusercontent.com/57274895/123248505-4c261f00-d4e8-11eb-87fb-4ce787fbeda8.png)

Reply:
When I try to do otherwise and wrap Dropdown to Link, there is no cursor pointer at all

Reply:
I wrapped Dropdown to Link and it is okay